package.json

```diff
@@ -12,8 +12,10 @@
   "dependencies": {
     "@types/canvasjs": "^1.9.11",
     "canvasjs": "^1.8.3",
+    "classnames": "^2.5.1",
     "moment": "^2.30.1",
     "moment-timezone": "^0.5.45",
+    "normalize-scss": "^8.0.0",
     "normalize.css": "^8.0.1",
```

Comment:
if we're using normalize-scss now, let's maybe remove normalize.css?

Reply:
ok, right